cardano/tx.go

```diff
@@ -40,15 +40,39 @@ func CreateTx(
 		})
 	}
 
+	tokenChange := []cardanowallet.TokenAmount{}
```

Comment:
use `tokenChange := []cardanowallet.TokenAmount(nil)`
and then remove bellow
```if len(tokenChange) == 0 {
		tokenChange = nil
	}
	```

Reply:
Put all this token calculation in separate function and use it for multisigfee also